packages/components/src/DateTimePickers/pickers/DatePicker/DatePicker.component.js

```diff
@@ -30,7 +32,11 @@ class DatePicker extends React.PureComponent {
 	}
 
 	isSelectedDate(date) {
-		return this.props.selectedDate !== undefined && isSameDay(this.props.selectedDate, date);
+		return (
+			(this.props.selectedDate !== undefined && isSameDay(this.props.selectedDate, date)) ||
+			this.isStartDate(date) ||
+			this.isEndDate(date)
```

Comment:
add startDate and endDate because they are supposed to have selected style.

Reply:
But this makes the start and end date focusable in the focus flow. That should not happen, only the current value should.
Please separate the date selection to the style.

Reply:
fix the style in another way